test/unit/runner.spec.js

```diff
@@ -119,9 +125,12 @@ describe('Runner', function () {
     });
 
     it('should not fail when a new common global is introduced', function () {
+      if (process.browser) {
```

Comment:
This test makes some assumptions about global properties that we can't guarantee across browsers.